tests/foreman/ui/test_contentview.py

```diff
@@ -1424,7 +1419,7 @@ def test_positive_remove_cv_version_from_default_env(session, module_org):
 
 @pytest.mark.tier2
 @pytest.mark.skipif((not settings.robottelo.REPOS_HOSTING_URL), reason='Missing repos_hosting_url')
-def test_positive_remove_promoted_cv_version_from_default_env(session, module_org):
+def test_positive_remove_promoted_cv_version_from_default_env(session, module_org, default_sat):
```

Comment:
This should be target_sat. May be other instances of default_sat left in this pr to swap out.